apps/frontend/src/components/observations/plant-observation.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { PlantInformation } from "../../../types/observations";
+interface PlantObservationProps {
+  plantInfo: PlantInformation;
+  editingButtons?: boolean;
+}
+
+const PlantObservation: React.FC<PlantObservationProps> = ({ plantInfo, editingButtons }) => {
```

Comment:
nit: Why the switch to functional components? Keep them all the same for consistency